stanford_fellowship.ds.inc

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * @file
+ * stanford_fellowship.ds.inc
+ */
+
+/**
+ * Implements hook_ds_layout_settings_info().
+ */
+function stanford_fellowship_ds_layout_settings_info() {
+  $export = array();
+
+  $ds_layout = new stdClass();
+  $ds_layout->api_version = 1;
+  $ds_layout->id = 'node|stanford_fellowship|full';
```

Comment:
Here is the line where the 'Full content' view mode is set to disabled by default: https://github.com/SU-SWS/stanford_fellowship/pull/1/files#diff-d6a2bb0339a1b091b37ace1e610c9178L72